gnomad/resources/grch38/gnomad.py

```diff
@@ -290,6 +290,15 @@
 )
 
 
+def get_coverage_ht(coverage_ht, data_type, coverage_version):
```

Comment:
If coverage_ht is already a table, do you need to read it again? You may want 3 options here:
1. auto (read the default coverage table for the type+version)
2. use the existing hl.Table instance, do nothing
3. read from a custom table URL

Reply:
oops yes that is a mistake (I never actually tried passing a coverage ht)

Reply:
should be fixed now